calcom/cal.com
# chore: removes unnecessary prisma env warnings

## What does this PR do?

Gets this rid of this warning that clogs CI logs and E2E tests. Making them tedious to debug and find useful data:

<img width="2056" alt="image" src="https://github.com/user-attachments/assets/6d268077-ef59-47d0-9eb2-3e466a7ed5ea">


- Updates the Prisma client to handle environment variable conflicts with a "none" check instead of a "warn" check. Until there's [a better alternative](https://github.com/prisma/prisma/issues/9726).

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- Verify that no warnings are displayed related to environment variable conflicts when using the Prisma client.
- Test various Prisma client operations to ensure they function correctly with the updated conflict check.

## Checklist

- I have read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code follows the style guidelines of this project
- I have commented my code, particularly in hard-to-understand areas
- I have checked if my changes generate no new warnings
